### ISSUE TITLE
Prices are not balanced

### ISSUE THREAD

**Issue by mwerezak** [BahamutoD/BDArmory#23]
Not an urgent issue, but something that should probably be addressed sometime.

In particular, radomes are dirt cheap, when really they probably should be one of the most expensive parts on an aircraft. As a result there is no incentive to make radarless aircraft (such as ground attack aircraft or cheap fighters with IR missiles that link to a search radar for detection) since even if you have no radar guided weapons there's practically no cost.

**Comment by boomchacle** [comment]
Makes sense for a compact radar to be much more expensive than a fuel tank...
